layouts/shortcodes/name-list.html

```diff
@@ -0,0 +1,23 @@
+<div>
+    {{ $names := index $.Site.Data.names (.Get 0) }}
+
+    {{ $sortedNames := slice }}
+    {{ range $names.list }}
+        {{ $nameParts := split .name " " }}
+        {{ $lastName := index (last 1 $nameParts) 0 }}
+        {{ $newEntry := merge . (dict "lastName" $lastName) }}
+        {{ $sortedNames = $sortedNames | append $newEntry }}
+    {{ end }}
```

Comment:
Hey @bdfinst, I've added logic here to split names and order by the last word for each `name` entry.

This will put both signatories and contributors in alphabetical order by last name. It will order names with prefixes such as `van Gogh` using the last word the but currently don't see any like that in the list. Is this problematic? 

Alternatively, how would you feel about splitting `signatories` names into first/last and changing this `name-list` to order by last?

Reply:
We have the contributors in alpha sequence to make everyone equal. The signatures are in order of time. Not perfect, but for reasons.

BTW, merging the title-centering PR messed up the star button. There was a merge conflict that I tried resolving that probably broke it. I can dig in later to try to fix it unless you can ID the problem faster.

Reply:
I've submitted a PR to fix the star button 👍.

This PR also puts signatures in alphabetical order by last name too. If that's not a desired state, feel free to close this PR or outline any specific changes you'd like done.

Reply:
We have the signatures in time sequence to highlight some of the early adopters during talks.